marimo/_plugins/ui/_impl/dataframes/transforms/handlers.py

```diff
@@ -409,6 +411,198 @@ def handle_expand_dict(
 
 
+class IbisTransformHandler(TransformHandler["ibis.Table"]):
+    @staticmethod
+    def supports_code_sample() -> bool:
+        return False
+
+    @staticmethod
+    def handle_column_conversion(
+        df: "ibis.Table", transform: ColumnConversionTransform
+    ) -> "ibis.Table":
+        import ibis
+
+        if transform.errors == "ignore":
+            try:
+                # Use coalesce to handle conversion errors
+                return df.mutate(
+                    **{
+                        transform.column_id: ibis.coalesce(
+                            df[transform.column_id].cast(
+                                ibis.dtype(transform.data_type)
+                            ),
+                            df[transform.column_id],
+                        )
+                    }
+                )
+            except ibis.common.exceptions.IbisTypeError:
+                return df
+        else:
+            # Default behavior (raise errors)
+            return df.mutate(
+                **{
+                    transform.column_id: df[transform.column_id].cast(
+                        ibis.dtype(transform.data_type)
+                    )
+                }
+            )
+
+    @staticmethod
+    def handle_rename_column(
+        df: "ibis.Table", transform: RenameColumnTransform
+    ) -> "ibis.Table":
+        return df.rename({transform.new_column_id: transform.column_id})
+
+    @staticmethod
+    def handle_sort_column(
+        df: "ibis.Table", transform: SortColumnTransform
+    ) -> "ibis.Table":
+        return df.order_by(
+            [
+                df[transform.column_id].asc()
+                if transform.ascending
+                else df[transform.column_id].desc()
+            ]
+        )
+
+    @staticmethod
+    def handle_filter_rows(
+        df: "ibis.Table", transform: FilterRowsTransform
+    ) -> "ibis.Table":
+        import ibis
+
+        filter_conditions: list[ir.BooleanValue] = []
+        for condition in transform.where:
+            column = df[str(condition.column_id)]
+            value = condition.value
+            if condition.operator == "==":
+                filter_conditions.append(column == value)
+            elif condition.operator == "!=":
+                filter_conditions.append(column != value)
+            elif condition.operator == ">":
+                filter_conditions.append(column > value)
+            elif condition.operator == "<":
+                filter_conditions.append(column < value)
+            elif condition.operator == ">=":
+                filter_conditions.append(column >= value)
+            elif condition.operator == "<=":
+                filter_conditions.append(column <= value)
+            elif condition.operator == "is_true":
+                filter_conditions.append(column == True)  # noqa: E712
+            elif condition.operator == "is_false":
+                filter_conditions.append(column == False)  # noqa: E712
```

Comment:
Similar to the `True` case, you can negate using `~`:

```suggestion
                filter_conditions.append(~column)
```